Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php

```diff
@@ -32,6 +32,11 @@ public function testFixProvider()
         $template = '<?php %s';
         $tests = array();
         foreach (array('require', 'require_once', 'include', 'include_once') as $statement) {
+            $test[] = array(
+                sprintf('<?php %s "foo.php" ?>', $statement),
+                sprintf('<?php %s("foo.php") ?>', $statement),
```

Comment:
It duplicates this:
https://github.com/SpacePossum/PHP-CS-Fixer/blob/1_12_feature_IncludeFixer_add_missing_test_case/Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php#L67